app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -130,6 +154,28 @@ private fun ImportFilesScreen(
     )
 }
 
+@Composable
+private fun ImportTextFields(initialShouldShowEmailAddressesFields: Boolean) {
```

Comment:
I don't think this boolean has anything to do with an "initial" value

Reply:
Yeah it will be rework in the next PR with the states, for now it's temporary code for integration